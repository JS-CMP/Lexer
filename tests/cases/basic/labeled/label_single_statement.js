test: console.log("single statement");

