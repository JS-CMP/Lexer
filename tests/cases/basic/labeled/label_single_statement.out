#include "types/JsAny.hpp"
#include "global/global.hpp"
#include "types/objects/Types.hpp"
#include "customOperators/CustomOperators.hpp"

int main() {
test:
        console[u"log"](JS::Any(u"single statement"));
    return 0;
}
